### PR TITLE
Include stack trace with errors

### PR DESCRIPTION
**Pull request by SferaDev**
Closes: https://github.com/xataio/client-ts/issues/152

### CODE REVIEW

File: test/integration.test.ts [hunk 1]
```diff
@@ -461,12 +476,37 @@ describe('integration tests', () => {
...
 
-    // @ts-expect-error
-    expect(client.db.users.createMany(invalidUsers)).rejects.toMatchInlineSnapshot(`
+    expect(client.db.users.createMany(invalidUsers)).rejects.toMatchInlineSnapshot(
+      {
+        message: 'Unknown error',
+        status: 400,
+        errors: [
+          {
+            //message: 'record ignored: transaction failed',
```
Comment (SferaDev) [inline test/integration.test.ts:487]: To be updated when change in production is deployed

**Review by gimenete — APPROVED** [review]
LGTM!

Only downside of this solution I can see is that you cannot do `instanceof Error`. Not sure if that's a big deal. WDYT

**Comment by gimenete** [comment]
> The bad thing about the Error class instance is that it only has string messages.

I guess we could extend the Error class. WDYT?

**Comment by gimenete** [comment]
@SferaDev looks great!

**Pull request merged**